frontend/src/lib/components/icons.tsx

```diff
@@ -1415,3 +1415,65 @@ export function IconUploadFile(props: React.SVGProps<SVGSVGElement>): JSX.Elemen
         </svg>
     )
 }
+
+export function CheckCircleOutlined(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+            <path
+                fill="currentColor"
+                d="M10 0C4.48 0 0 4.48 0 10C0 15.52 4.48 20 10 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 10 0ZM10 18C5.59 18 2 14.41 2 10C2 5.59 5.59 2 10 2C14.41 2 18 5.59 18 10C18 14.41 14.41 18 10 18ZM14.59 5.58L8 12.17L5.41 9.59L4 11L8 15L16 7L14.59 5.58Z"
+            />
+        </svg>
+    )
+}
+
+export function CloseCircleOutlined(props: React.SVGProps<SVGSVGElement>): JSX.Element {
```

Comment:
You don't need to add this icon! It already exists as `IconCancel`. You can see it in the [Lemon UI Icons story](https://61260360a8e859003a7c138b-wrpovigrap.chromatic.com/?path=/docs/lemon-ui-icons--icons).

Reply:
Gahh, not sure how I missed all these existing ones! Good catches!